reference/twig_reference.rst

```diff
@@ -474,8 +474,43 @@ yaml_encode
 ``dumpObjects`` *(optional)*
     **type**: ``boolean`` **default**: ``false``
 
-Transforms the input into YAML syntax. See :ref:`components-yaml-dump` for
-more information.
+Transforms the input into YAML syntax.
+
+The ``inline`` argument is the level where you switch to inline YAML:
+
+.. code-block:: twig
+
+    {% set array = {
+        'a': {
+            'c': 'e'
+        },
+        'b': {
+            'd': 'f'
+        }
+    } %}
+
+    {{ array|yaml_encode(0) }}
```

Comment:
```suggestion
    {{ array|yaml_encode(inline = 0) }}
```

Reply:
I would use named arguments everywhere in `yaml_encode`/`yaml_dump` to make the examples more clear